GNUmakefile

```
@@ -291,8 +289,7 @@ ifeq (, $(shell which node))
 endif
 	@if [ $(NODEJS_VERSION) -lt $(MIN_NODEJS_VERSION) ]; then echo "Install NodeJS version 18+ to run tests."; exit 1; fi
 
-# Build the Go compiler.
-tinygo:
+tinygo: ## Build the Go compiler
```

nit: Maybe make it 
```suggestion
tinygo: ## Build the TinyGo compiler
```
to make a distinction from big go?

---

Done